packages/fxa-react/configs/tailwind.js

```diff
@@ -59,6 +59,10 @@ module.exports = {
         '-3': '-0.75rem',
         '-52': '-13rem',
         '55': '13.75rem',
+        ten: '10%',
+        full: '100%',
+        center: '44%',
```

Comment:
I have hesitation about such a specific number. It looks like this is used to horizontally center the caret/triangle on the tooltip, and while it looks visibly center it's actually slightly off in practice:

![Group](https://user-images.githubusercontent.com/6392049/93904855-48027200-fcc8-11ea-8c8c-47f44fa4a7b9.png)

Instead of this additional `left-center` class we could used `left-1/2 transform -translate-x-1/2` which should take care of perfectly centering the caret.